cqf-fhir-cr/src/main/java/org/opencds/cqf/fhir/cr/measure/r4/R4MeasureScoringTypePopulations.java

```diff
@@ -0,0 +1,299 @@
+package org.opencds.cqf.fhir.cr.measure.r4;
+
+import java.util.EnumSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+import org.opencds.cqf.fhir.cr.measure.common.MeasurePopulationType;
+import org.opencds.cqf.fhir.cr.measure.common.MeasureScoring;
+
+public class R4MeasureScoringTypePopulations {
+    public enum PROPORTION_ALLOWED {
+        INITIALPOPULATION(MeasurePopulationType.INITIALPOPULATION),
+        DENOMINATOR(MeasurePopulationType.DENOMINATOR),
+        DENOMINATOREXCLUSION(MeasurePopulationType.DENOMINATOREXCLUSION),
+        DENOMINATOREXCEPTION(MeasurePopulationType.DENOMINATOREXCEPTION),
+        NUMERATOREXCLUSION(MeasurePopulationType.NUMERATOREXCLUSION),
+        NUMERATOR(MeasurePopulationType.NUMERATOR),
+        DATEOFCOMPLIANCE(MeasurePopulationType.DATEOFCOMPLIANCE),
+        TOTALDENOMINATOR(MeasurePopulationType.TOTALDENOMINATOR),
+        TOTALNUMERATOR(MeasurePopulationType.TOTALNUMERATOR);
+
+        private final MeasurePopulationType measurePopulationType;
+
+        PROPORTION_ALLOWED(MeasurePopulationType measurePopulationType) {
+            this.measurePopulationType = measurePopulationType;
+        }
+
+        public MeasurePopulationType getPopulationType() {
+            return measurePopulationType;
+        }
+
+        public static Set<MeasurePopulationType> getPopulations() {
+            EnumSet<R4MeasureScoringTypePopulations.PROPORTION_ALLOWED> data = EnumSet.allOf(PROPORTION_ALLOWED.class);
+            return data.stream().map(PROPORTION_ALLOWED::getPopulationType).collect(Collectors.toSet());
+        }
+
+        public static void isMember(List<MeasurePopulationType> populations) {
```

Comment:
Suggest you rename this method to "validateMember()".  "is" implies you're returning a boolean.